src/chainparams.cpp

```diff
@@ -131,9 +131,9 @@ class CMainParams : public CChainParams
         bnProofOfWorkLimit = ~uint256(0) >> 20; // PIVX starting difficulty is 1 / 2^12
         nSubsidyHalvingInterval = 210000;
         nMaxReorganizationDepth = 100;
-        nEnforceBlockUpgradeMajority = 750;
-        nRejectBlockOutdatedMajority = 950;
-        nToCheckBlockUpgradeMajority = 1000;
+        nEnforceBlockUpgradeMajority = 1080;
+        nRejectBlockOutdatedMajority = 1368;
+        nToCheckBlockUpgradeMajority = 1440;
```

Comment:
What is the rationale behind this change? I know that the percentages are unaffected, just looking for the reasoning to extend the block window.

Reply:
There are 1440 blocks per day. Numbers changed to be in line with a lookback period of at least one day.

Reply:
Ok, that’s what I thought, but didn’t want to take that assumption